spec/integration/rails/graphql/relay/relation_connection_spec.rb

```diff
@@ -93,10 +101,9 @@ def get_last_cursor(result)
       assert_equal(true, conn["pageInfo"]["hasNextPage"])
 
       log_entries = log.split("\n")
-      assert_equal 2, log_entries.size, "It ran 2 sql queries"
+      assert_equal 1, log_entries.size, "It should run 1 sql query"
```

Comment:
The change seems to remove the need for the following query:

`SELECT 1 AS one FROM bases WHERE bases.id IN (SELECT bases.id FROM bases WHERE bases.faction_id = ?) LIMIT ? OFFSET ?\e[0m  [[faction_id, 2], [LIMIT, 1], [OFFSET, 2]`